pages/dojo.md

```diff
@@ -0,0 +1,25 @@
+title: Code Dojo
+---
+
+On Thursday evening we'll be running a Python Code Dojo, based on the long-running
+[London Python Code Dojo](http://ldnpydojo.org.uk/) series, now approaching its 10th year.
+
+It's a very welcoming format where a coding challenge is set and everyone takes
+part as one of a team. It's definitely _not_ a competition, and it definitely
+_is_ a place where beginners or less experienced coders can join in.
+
+The layout of the evening is as follows:
+
+* We eat, drink and socialise
+* We choose a challenge
```

Comment:
Except in this case we don't. Perhaps change to:

"We usually choose a challenge, but in this case we're going write bots to play each other at this Python game: https://robotgame.net/"

Reply:
Done